tests/callbacks/test_callbacks.py

```diff
@@ -224,8 +222,7 @@ def training_step(self, *args, **kwargs):
             output.update({'my_train_metric': loss})
             return output
 
-    hparams = tutils.get_default_hparams()
-    model = ModelWithoutValStep(hparams)
+    model = ModelWithoutValStep(tutils.get_default_hparams())
```

Comment:
@Borda we need to get rid of these models. the only one to use is the template model...

Reply:
This is refactor only for default, the others are in other PRs per test package